### PR TITLE
Video Component created

### PR DESCRIPTION
**Pull request by Naveennn2358**

### CODE REVIEW

**Comment by Naveennn2358** [comment]
Please Merge the code

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/videocomponent.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<sly data-sly-use.compo="com.adobe.aem.guides.glucerna.core.models.VideoComp"
+     data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+     data-sly-test.empty="${compo.damVideoPath || compo.youtubeUrl}">
+    <div data-sly-test="${compo.path == 'dam'}">
+        <div class="video-container video_dam">
+            <video id="myVideo"
+                   poster="/content/dam/glucerna/glucerna%20videoposter.jpg"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/videocomponent.html]: Don't hard code the path. Use configuration in dialog box.

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/videocomponent.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                   >
+                <source src="${compo.damVideoPath}" type="video/mp4" />
+            </video>
+            <div class="play-button" id="playButton"></div>
+        </div>
+    </div>
+    <div data-sly-test="${compo.path == 'youtube'}">
+        <div class="video-container vid_youtube">
+            <iframe style="width:100%;height:100%;"
+                    src="${compo.youtubeUrl}?autoplay=1&rel=0&mute=1"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/videocomponent.html]: Don't hard code the autoplay value make it configurations.

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/_cq_dialog/.content.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    <dam
+                        granite:class="showhidetargets"
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <dampath
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/pathbrowser"
+                                fieldLabel="DAM Path"
+                                name="./damVideoPath"/>
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/videocomponent/_cq_dialog/.content.xml]: Provide the dam path in root.

**Pull request merged**